src/config/datachart.json

```diff
@@ -498,22 +498,6 @@
                         "jp": "$.ig.DataSource からデータを要求するには、$.ig.DataSource により承諾されたリモート URL を指定します。"
                     }
                 },
-                "dataSourceType": {
```

Comment:
If the user setup uses remote datasource with remote url, is this property required?